### ISSUE TITLE
Update to Poison >= 3

### ISSUE THREAD

**Issue by mrusme** [danielberkompas/ex_twilio#72]
`mix deps.get` is complaining locally, that ex_twilio requires an older version of Poison:

```
Running dependency resolution...

Failed to use "poison" (version 3.1.0) because
  ecto (version 2.1.4) requires ~> 2.2 or ~> 3.0
  ex_twilio (version 0.3.0) requires ~> 2.2
  maru (version 0.11.4) requires ~> 1.5 or ~> 2.0 or ~> 3.0
  mix.lock specifies 3.1.0

** (Mix) Hex dependency resolution failed, relax the version requirements of your dependencies or unlock them (by using mix deps.update or mix deps.unlock). If you are unable to resolve the conflicts you can try overriding with {:dependency, "~> 1.0", override: true}
```

Of course one can override with `{:poison,       "~> 3.1", override: true}`, but it would be nicer, if ex_twilio would get upgraded to support ~> 3.0.

**Comment by pablo-co** [comment]
+1

**Comment by andrewshatnyy** [comment]
I think this is addressed already https://github.com/danielberkompas/ex_twilio/commit/1f1e35b2c4408e431a642ec2460ff28c0c8b72ea#diff-6023be6004fce4718dad3dafb576d258R23

**Comment by mrusme** [comment]
@andrewshatnyy nice, thank you! Now all what's missing is an actual release of the latest ex_twilio code in order to be able to benefit from it. :-)

**Comment by danielberkompas** [comment]
I've released a new version: `0.4.0`. Sorry for the delay, guys.

**Comment by mrusme** [comment]
@danielberkompas thank you Sir!
